about.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>London Love</title>
```

Comment:
Might want to put your name in the title so it shows up when people google ya once you host it